github-actions/oss-readme-template/__tests__/readme.test.ts

```diff
@@ -52,11 +67,170 @@ STATIC CONTENT
     expect(() => {
       generateReadmeStringFromTemplateString({
         templateContents: templateStringWithoutOssHeader,
+        projectInfo: {type: ProjectType.OTHER},
         projectStatus: ProjectStatus.INCUBATING,
         projectStability: ProjectStability.EXPERIMENTAL,
       });
     })
       .toThrowError(`README template does not conform to Momento OSS requirements:
-{"lineNumber":2,"ruleNames":["must-begin-with-oss-header"],"ruleDescription":"Template must begin with OSS Header","ruleInformation":"https://github.com/momentohq/standards-and-practices/github-actions/oss-readme-generator","errorDetail":"Expected template file to begin with {{ ossHeader }}, on a line by itself.","errorContext":null,"errorRange":null,"fixInfo":null}`);
+{"lineNumber":2,"ruleNames":["must-include-oss-headers"],"ruleDescription":"Template must begin with OSS Header and end with OSS Footer","ruleInformation":"https://github.com/momentohq/standards-and-practices/github-actions/oss-readme-generator","errorDetail":"Expected template file to begin with {{ ossHeader }}, on a line by itself.","errorContext":null,"errorRange":null,"fixInfo":null}
+{"lineNumber":2,"ruleNames":["must-include-oss-headers"],"ruleDescription":"Template must begin with OSS Header and end with OSS Footer","ruleInformation":"https://github.com/momentohq/standards-and-practices/github-actions/oss-readme-generator","errorDetail":"Expected template file to end with {{ ossFooter }}, on a line by itself.","errorContext":null,"errorRange":null,"fixInfo":null}`);
+  });
+
+  const VALID_TEMPLATE_CONTENTS = fs
+    .readFileSync(path.join(__dirname, 'workflows', 'valid-sdk-template.md'))
+    .toString();
+
+  const EXAMPLE_SDK_PROJECT_INFO: SdkProject = {
+    type: ProjectType.SDK,
+    language: 'WaterLoop',
+  };
+
+  it('succeeds for an SDK README that includes all of the expected section headers', () => {
+    expect(
+      generateReadmeStringFromTemplateString({
+        templateContents: VALID_TEMPLATE_CONTENTS,
+        projectInfo: EXAMPLE_SDK_PROJECT_INFO,
+        projectStatus: ProjectStatus.OFFICIAL,
+        projectStability: ProjectStability.STABLE,
+      })
+    )
+      .toEqual(`<img src="https://docs.momentohq.com/img/logo.svg" alt="logo" width="400"/>
+
+[![project status](https://momentohq.github.io/standards-and-practices/badges/project-status-official.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md)
+[![project stability](https://momentohq.github.io/standards-and-practices/badges/project-stability-stable.svg)](https://github.com/momentohq/standards-and-practices/blob/main/docs/momento-on-github.md) 
+
+# Momento WaterLoop Client Library
```

Comment:
`WaterLoop`? Sorry I missed this before maybe what is this exactly? Never heard term before.

Reply:
Is it just supposed to be a hypothetical lang?